sdk_container/src/third_party/coreos-overlay/coreos/config/env/sys-auth/sssd

```diff
@@ -1 +1,11 @@
-export ac_cv_member_struct_ldap_conncb_lc_arg=no
```

Comment:
Wouldn't it be better to just keep this instead of adding the patch? Or if you're confident about this check being redundant, I could just add this line to Gentoo.

Reply:
I wanted to switch it to "yes", because our version of OpenLDAP has the `lc_arg` member in `struct ldap_conncb` and using it seems to allow SSSD to track some referrals or something (in general this seems to be preferred). But the "yes" case has a runtime check for a bug that was fixed 16 years ago. And we know that runtime checks and cross-compilation do go along.

I think that long-term my preferences would be in this order:

1. Upstream to drop the runtime check, just like my user-patch is doing.
2. Upstream adding a `--enable-ldap-conncb-i-know-what-i-am-doing` flag that skips the runtime check.
3. Gentoo taking my patch.

Reply:
Ah, I had it backwards. In that case, a better option may be to populate the 4th `AC_RUN_IFELSE` argument, which is currently `[]`, to simply assume `HAVE_LDAP_CONNCB=1` when cross-compiling. This seems like a reasonable compromise given how old that release is now. Upstream may be more willing to do that than drop the check entirely.

Reply:
I have filed a PR to SSSD, let's see how it goes - https://github.com/SSSD/sssd/pull/7743